code/modules/reagents/reagents/other.dm

```diff
@@ -401,11 +401,12 @@
 	overdose_threshold = REAGENTS_OVERDOSE
 	overdose_crit_threshold = REAGENTS_OVERDOSE_CRITICAL
 	taste_description = "gross metal"
+	var/obj/effect/effect_type = /obj/effect/decal/cleanable/liquid_fuel
```

Comment:
Autodoc, also dont get why this is cast if its just a normal typepath without initial()

Reply:
I was going to do something different, forgot to change it after I just did it this way.